debian/control

```diff
@@ -10,8 +10,7 @@ Package: linux-firmware
 Architecture: all
 Multi-Arch: foreign
 Priority: optional
-Depends: ${misc:Depends}
-Recommends: firmware-sof-signed
```

Comment:
We normal handle desired recommends by installing them explicitly in eos-meta, no?

Reply:
Uh!  Okay!